src/template_storage.mligo

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2021 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+// This type should define the type of the contract-extra field in storage
+// for this variant.
+type contract_extra = (string, bytes) big_map
+
+// This should define a default value for the contract-extra field for the
+// variant.
+let default_extra : contract_extra = (Big_map.empty : contract_extra)
```

Comment:
Why define the standard as a `Big_map`, I think using `unit` (as in `trivialDAO`) would make this less of a burden in case someone is implementing a custom DAO and forgets to update these types (because they don't need any extra).

What do you think?

Reply:
Changed to unit.